avalon_listener/listener/tcs_listener.py

```diff
@@ -259,36 +261,49 @@ def parse_command_line(config, args):
         config['Logging']['LogLevel'] = options.loglevel.upper()
     if options.bind_uri:
         bind_uri = options.bind_uri
+    else:
+        bind_uri = config["Listener"]["listener_port"]
     if options.lmdb_url:
-        config["KvStorage"]["remote_url"] = options.lmdb_url
+        config["KvStorage"]["remote_storage_url"] = options.lmdb_url
 
     return bind_uri
 
 # -----------------------------------------------------------------
 # -----------------------------------------------------------------
 
 
+def get_config_dir():
+    """
+    Returns the avalon configuration directory based on the
+    TCF_HOME environment variable (if set) or OS defaults.
+    """
+    if 'TCF_HOME' in os.environ:
+        return os.path.join(os.environ['TCF_HOME'], 'avalon_listener/')
+
+    return '/etc/avalon'
```

Comment:
It's easy to manage if we make it mandatory to set TCF_HOME directory

Reply:
Once we have the debians, TCF_HOME is not used.

Reply:
using TCF_HOME/AVLON_HOME looks better option than using hard coded path /etc/avalon

Reply:
Directory `/etc` is reserved for system and configuration files. But it is not an directory to install software.  A more standard default would be `/opt/avalon`

I am OK with making `TCF_HOME` mandatory and failing if it is not set.

Reply:
Since this module is for avalon listener, it looks more apt to name home directory something like AVALON_LISTENER_HOME